### ISSUE TITLE
Forked PR's fail CI

### ISSUE THREAD

**Issue by wlach** [iodide-project/iodide#2330]
Since we enabled codecov, it looks like forked PRs are now failing the test-server job with this:

```
-> Pinging Codecov
https://codecov.io/upload/v4?package=bash-8a28df4&token=secret&branch=pull%2F2329&commit=e84326b038576f73971c632c3561be13ff94b607&build=&build_url=&name=&tag=&slug=iodide-project%2Fiodide&service=&flags=&pr=&job=
HTTP 400
Please provide the repository token to upload reports via `-t :repository-token`
Exited with code 1
```

See e.g. #2329

I think this is because the `CODECOV_TOKEN` variable is private to iodide. There is an option in circle [to pass secrets from forked PRs](https://circleci.com/docs/2.0/oss/#pass-secrets-to-builds-from-forked-pull-requests), but we probably don't want to blanket enable that: I don't know how sensitive the `CODECOV_TOKEN` is, but there are dockerhub credentials which we definitely don't want to leak here.

@jezdez do you have any ideas on how to proceed? [This page](https://docs.codecov.io/docs/about-the-codecov-bash-uploader) suggests that we might not even need a token?

![image](https://user-images.githubusercontent.com/20569/66164588-0123f400-e601-11e9-887a-9cdb530a9226.png)

**Comment by wlach** [comment]
So I went ahead and fixed this, but the approach

**Comment by wlach** [comment]
I resolved this in a hacky way in #2332. Filed a bug about finding a better solution in #2333.
